packages/karma-ui5-transpile/lib/index.js

```diff
@@ -75,21 +74,23 @@ function createPreprocessor(config, logger) {
 				return plugin.file.request === "istanbul";
 			})
 		) {
+			const istanbulConfig = {
+				include: ["**/*"],
+				exclude: []
+			};
+			// apply the `instrumenterOptions` for istanbul from `coverageReporter` in the `karma.config`
+			const instrumenterOptionsIstanbul = config.coverageReporter?.instrumenterOptions?.istanbul;
+			if (instrumenterOptionsIstanbul && typeof instrumenterOptionsIstanbul === "object") {
```

Comment:
A simple `typeof` check is sufficient in this case as `typeof undefined === "object"` => `false`

Reply:
Yes, I somehow had the misconception in my head, that optional chaining would yield `null` instead of `undefined`. Checked documentation and it looks like I was wrong. (maybe I got that from some strange polyfills I used for it in the past)